geomstats/learning/k_means.py

```diff
@@ -0,0 +1,87 @@
+import random
+
+import geomstats.backend as gs
+from geomstats.learning._template import TemplateTransformer
+
+
+class KMeans(TemplateTransformer):
```

Comment:
I'm not sure you should inherit from `TemplateTransformer` here. I think the TemplateTransformer class was written to give you an example of what a child class of TransformerMixin should look like.

In TemplateTransformer, there is 

```class TemplateTransformer(BaseEstimator, TransformerMixin):```

and in the Kmean code, there is:
https://github.com/scikit-learn/scikit-learn/blob/b194674c4/sklearn/cluster/_kmeans.py#L649

```class KMeans(TransformerMixin, ClusterMixin, BaseEstimator):```

thus, how about inheriting from TransformerMixin, ClusterMixin, BaseEstimatorm, ie:

```class RiemannianKMeans(TransformerMixin, ClusterMixin, BaseEstimator):```